package.json

```diff
@@ -3,16 +3,14 @@
   "version": "14.0.0",
   "description": "Format whitespace in a SQL query to make it more readable",
   "license": "MIT",
-  "main": "dist/sql-formatter.min.cjs",
+  "main": "dist/index.cjs",
```

Comment:
This now uses the un-minified, non-UMD bundle. I think this is ok; `unpkg` will continue using the minified+UMD bundle produced by webpack.